app/models/move.rb

```diff
@@ -90,6 +113,11 @@ class Move < VersionedModel
 
   scope :not_cancelled, -> { where.not(status: MOVE_STATUS_CANCELLED) }
 
+  scope :updated_at_from_and_to, lambda { |from, to|
```

Comment:
maybe `updated_range` is easier to remember/to work with?

Reply:
Feexed